students/GuyCChan/session08/test_circleClass.py

```diff
@@ -0,0 +1,150 @@
+from circleClass import Circle, Sphere
+import pytest
+from math import pi
+
+# Test for initialization of the class.
+def test_init():
+    Circle(4)
+
+def test_radius():
+    c = Circle(4)
+    assert c.radius == 4
+    assert c.radius != 6
+
+def test_diameter():
+    c = Circle(4)
+    assert c.diameter == 8
+    assert c.diameter != 10
+
+def test_radius_reset():
+    c = Circle(4)
+    c.radius = 10
+    assert c.radius == 10
+    assert c.diameter == 20
+
+def test_diameter_set():
+    c = Circle(10)
+    c.diameter = 8
+    assert c.diameter == 8
+    assert c.radius == 4
+    assert c.radius != 10
+
+def test_area():
+    c = Circle(5)
+    assert c.area == pi * 5 **2
+    assert c.area != 80
+
+def test_area_from_diam():
+    c = Circle(5)
+    assert c.area == pi * 5 ** 2
+
+def test_area_from_diameter():
```

Comment:
tests aer good, but the names aren't -- you are simply testing the area property...